client/components/JobList/JobList.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import JobCard from '../jobCard/jobCard';
+
+export default function JobList() {
+  const [jobs, setJobs] = useState([]);
+  const [error, setError] = useState('');
+  useEffect(() => {
+    getJobs();
+  }, []);
+  //break this out into it's own hook maybe?
+  function getJobs() {
+    fetch('/api/v1/jobs').then((response) => {
+      return response.json().then((data) => {
+        console.log('DATA IS: ', data);
+        setJobs(data);
+      });
+    });
```

Comment:
I agree with your comment, I think you should build out your fetch out in a services file and then import it into this component. Looks great otherwise